docsets/PyMongo/docset.json

```diff
@@ -1,15 +1,19 @@
 {
     "name": "PyMongo",
-    "version": "3.9.0",
+    "version": "3.10.1",
     "archive": "PyMongo.tgz",
```

Comment:
You need to update the archive at this path as well.

Reply:
Done. https://github.com/Kapeli/Dash-User-Contributions/pull/2675/commits/ec505531b52d5aea60fc5bd708eaaeb7770a44aa